src/libwalletqt/WalletManager.cpp

```diff
@@ -186,6 +182,11 @@ Wallet *WalletManager::createWalletFromDevice(const QString &path, const QString
 {
     QMutexLocker locker(&m_mutex);
     WalletPassphraseListenerImpl tmpListener(this);
+    m_passphraseReceiver = &tmpListener;
```

Comment:
Please guard the assignment the `m_mutex_passphraseReceiver` too

Reply:
ah, true, done

Reply:
Is okay now